### PR TITLE
INFRA-2366 Jenkins admin chatbot to AKS

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: clusters/publick8s.yaml [hunk 1]
```diff
@@ -18,3 +18,4 @@ helmfiles:
   - path: "../helmfile.d/oauth2-proxy.yaml"
   - path: "../helmfile.d/release-jenkins.yaml"
   - path: "../helmfile.d/plugin-site.yaml"
+  - path: "../helmfile.d/chatbot_jenkinsadmin.yaml"
```
Comment (timja) [inline clusters/publick8s.yaml]: is this the right cluster? wasn't sure?

File: CODEOWNERS [hunk 2]
```diff
@@ -4,5 +4,6 @@
 
 jenkins-wiki-exporter/ @halkeye
 plugin-site/ @timja
+chatbot_jenkinsadmin/ @timja
```
Comment (olblak) [inline CODEOWNERS]:
```suggestion
chatbot_jenkinsadmin/ @timja @slide
```

Comment (olblak) [reply CODEOWNERS]: @slide as he regularly contributes to this bot

File: charts/chatbot_jenkinsadmin/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for plugins.jenkins.io
+name: jenkinsadmin
+maintainers:
+  - name: timja
```
Comment (olblak) [inline charts/chatbot_jenkinsadmin/Chart.yaml]:
```suggestion
  - name: timja
  - name: slide
```

File: charts/chatbot_jenkinsadmin/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Plugin site
```
Comment (olblak) [inline charts/chatbot_jenkinsadmin/README.md]:
```suggestion
# Chatbot Jenkinsadmin
```

**Comment by olblak** [comment]
Before merging this PR, I would like to see if we can use github oauth token instead of github username/password, especially because the current user used has too many permission

**Comment by timja** [comment]
> Before merging this PR, I would like to see if we can use github oauth token instead of github username/password, especially because the current user used has too many permission

I've never hit somewhere that I've not been able to use an oauth token on GitHub, it's required to work in general because a lot of orgs require 2fa to be enabled which prevents using a username and password

**Comment by olblak** [comment]
> I've never hit somewhere that I've not been able to use an oauth token on GitHub, it's required to work in general because a lot of orgs require 2fa to be enabled which prevents using a username and password
Let rephrase this, how easy would it be to modify jenkins-infra/ircbot to use github token instead of username/password :D I created INFRA-2371

**Comment by timja** [comment]
Shouldn't need any work at all, just use an oauth token instead of password

**Comment by olblak** [comment]
let's try

**Pull request merged**